core/src/test/resources/importSingleLine/ImportSingleLine.source

```diff
@@ -20,3 +20,14 @@ import org.{
 }
 >>>
 import org.{Aaaa, Bbbbb, C, D}
+<<< handle inline comments
+import org.{
+// what?
+  Aaaa,
+  Bbbbb,
+  C,
+  D
+}
+>>>
+import org.{// what?
```

Comment:
I think this is totally fine, the user can move the comment elsewhere and re-format.